scripts/Start.ps1

```diff
@@ -8,6 +8,18 @@ $FrontendScript = Join-Path "$PSScriptRoot" 'Start-Frontend.ps1'
 
 # Start backend (in new PS process)
 Start-Process pwsh -ArgumentList "-command $BackendScript"
+# check if the backend is running before proceeding
+$backendRunning = $false
+
+# get the port from the REACT_APP_BACKEND_URI env variable
+$envFilePath = Join-Path $PSScriptRoot '..\webapp\.env'
+$envContent = Get-Content -Path $envFilePath
+$port = [regex]::Match($envContent, ':(\d+)/').Groups[1].Value
+
+while ($backendRunning -eq $false) {
```

Comment:
Would be a good idea to add a timeout in case the backend server goes dark for any reason

Reply:
Agreed. I just found the SK Office Hours on YouTube. Very helpful! Also, the Copilot series with Alex Chao ( @alexchaomander ) is very informative as well. 🙂

Reply:
Super happy to hear you found it helpful :) You're welcome to OH anytime, we host them every Wednesday.